applications/nrf_desktop/doc/led_state.rst

```diff
@@ -1,6 +1,54 @@
 .. _led_state:
 
-led_state module
+LED state module
 ################
 
-The ``led_state`` module is used to update the LED state based on the BLE connection state and the system state.
+The ``led_state`` module is used to set LED effects based on the Bluetooth peer state and the system state.
+
+Controlled LEDs
+***************
+
+The module controls LEDs defined by enumerators in :cpp:enum:`led_id`:
+
+* :cpp:enumerator:`LED_ID_SYSTEM_STATE` - shows system state. The system can be in one of the following states:
+
+  * :cpp:enumerator:`LED_SYSTEM_STATE_IDLE` - the device is not being charged,
+  * :cpp:enumerator:`LED_SYSTEM_STATE_CHARGING` - the device is being charged,
+  * :cpp:enumerator:`LED_SYSTEM_STATE_ERROR` - fatal application error occurred (a module reported error state or battery state error was reported).
+
+* :cpp:enumerator:`LED_ID_PEER_STATE` - shows Bluetooth peer state. The Bluetooth peer can be in one of the following states:
+
+  * :cpp:enumerator:`LED_PEER_STATE_DISCONNECTED` - the Bluetooth peer is disconnected,
+  * :cpp:enumerator:`LED_PEER_STATE_CONNECTED` - the Bluetooth peer is connected,
+  * :cpp:enumerator:`LED_PEER_STATE_CONFIRM_SELECT` - the Bluetooth peer is being selected (the device is waiting for confirmation),
+  * :cpp:enumerator:`LED_PEER_STATE_CONFIRM_ERASE` - the device is waiting for user confirmation to erase peers (for Bluetooth central) or start erase advertising (for Bluetooth peripheral),
+  * :cpp:enumerator:`LED_PEER_STATE_ERASE_ADV` - the device is advertising for peer erase.
+
+Complete description of peer management is available in :ref:`ble_bond`.
+
+Module Events
+*************
+
++------------------------+------------------------------+---------------+----------------+-------------+
+| Source Module          | Input Event                  | This Module   | Output Event   | Sink Module |
++========================+==============================+===============+================+=============+
+| :ref:`ble_bond`        | ``ble_peer_operation_event`` | ``led_state`` | ``led_event``  | :ref:`leds` |
++------------------------+------------------------------+               |                |             |
+| :ref:`ble_state`       | ``ble_peer_event``           |               |                |             |
++------------------------+------------------------------+               |                |             |
+| :ref:`battery_charger` | ``battery_state_event``      |               |                |             |
++------------------------+------------------------------+---------------+----------------+-------------+
+
+Configuration
+*************
+
+Module is enabled when ``CONFIG_DESKTOP_LED_ENABLE`` option is set (when :ref:`leds` is configured and enabled).
```

Comment:
```suggestion
The module is enabled when you set the ``CONFIG_DESKTOP_LED_ENABLE`` option, which is possible when the :ref:`leds` sink module is configured and enabled.
```

Reply:
It is possible to set option when `leds` module not configured, but this will cause errors. I will change this a bit